modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -68,12 +68,27 @@ definition {
 	}
 
 	@description = "LPS-144474 - Verify that the user, when modeling an Object, is able to add validations using Groovy"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanUserAddValidationsGroovy {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 153550",
+			objectName = "CustomObject153550",
+			pluralLabelName = "Custom Objects 153550");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 153550");
+
+		ObjectAdmin.goToValidationsTab();
 
-		// TODO LPS-153550 CanUserAddValidationsGroovy pending implementation
+		ObjectCustomValidation.addObjectValidation(
+			validationName = "Custom Validation",
+			validationType = "Groovy");
 
+		ObjectCustomValidation.assertValidationPresent(
+			validationActive = "No",
+			validationName = "Custom Validation",
+			validationType = "Groovy");
```

Comment:
Can we actually do the validation using Groovy for this test? Adding the validation is fine but it would be a better end-to-end test if we verify the functionality works. What do you think?